mne/viz/backends/_ipyvolume.py

```diff
@@ -0,0 +1,372 @@
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Oleh Kozynets <ok7mailbox@gmail.com>
+#          Guillaume Favelier <guillaume.favelier@gmail.com>
+#
+# License: Simplified BSD
+
+import warnings
+import numpy as np
+from .base_renderer import _BaseRenderer
+from ...utils import copy_base_doc_to_subclass_doc
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import ipyvolume as ipv
+    from pythreejs import (BlendFactors, BlendingMode, Equations,
+                           ShaderMaterial, Side)
+
+
+@copy_base_doc_to_subclass_doc
+class _Renderer(_BaseRenderer):
+    """Class managing rendering scene created with ipyvolume.
+
+    Attributes
+    ----------
+    plotter: ipyvolume.widgets.Figure
+        The scene handler.
+    """
+
+    def __init__(self, fig=None, size=(600, 600), bgcolor=(0., 0., 0.),
+                 name="Ipyvolume Scene", show=False):
+        """Set up the scene.
+
+        Parameters
+        ----------
+        fig: ipyvolume.widgets.Figure
+            The scene handler.
+        size: tuple
+            The dimensions of the context window: (width, height).
+        bgcolor: tuple
+            The color definition of the background: (red, green, blue).
+        name: str | None
+            The name of the scene.
+        """
+        self.off_screen = False
+        self.name = name
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=DeprecationWarning)
+            if fig is None:
+                fig_w, fig_h = size
+                self.plotter = ipv.figure(width=fig_w, height=fig_h)
+                self.plotter.animation = 0
+                ipv.style.box_off()
+                ipv.style.axes_off()
+                bgcolor = tuple(int(c) for c in bgcolor)
+                ipv.style.background_color('#%02x%02x%02x' % bgcolor)
+            else:
+                self.plotter = ipv.figure(key=fig)
+
+    def scene(self):
+        return self.plotter
+
+    def set_interactive(self):
+        pass
+
+    def mesh(self, x, y, z, triangles, color, opacity=1.0, shading=False,
+             backface_culling=False, **kwargs):
+        # opacity for overlays will be provided as part of color
+        color = _color2rgba(color, opacity)
+        mesh = ipv.plot_trisurf(x, y, z, triangles=triangles, color=color)
+        _add_transperent_material(mesh)
+        ipv.squarelim()
+
+        return mesh
+
+    def contour(self, surface, scalars, contours, line_width=1.0, opacity=1.0,
+                vmin=None, vmax=None, colormap=None):
+        from matplotlib import cm
+        from matplotlib.colors import ListedColormap
+        from ipyvolume.pylab import plot
+
+        vertices = surface['rr']
+        tris = surface['tris']
+
+        if colormap is None:
+            cmap = cm.get_cmap('coolwarm')
+        else:
+            cmap = ListedColormap(colormap / 255.0)
```

Comment:
It's not normalized in the current use case but I'll update the `_BaseRenderer` API and add a parameter `normalized=False`, thanks for the suggestion.

Reply:
what I meant was that if I understood the code properly, if I do this:

```py
colormap = _get_my_01_normalized_cmap()  # however should I do this because I don't find where is defined
```

then, my `[0,1]` color map becomes `[0, 0.003]`

Reply:
The expected range for `colormap` values was `[0, 255]` so far (dictated by the available 3d backends usage). Having an additionnal parameter with its docstring will help making it clear in my opinion.

In the current state of the code, the value will be considerered unnormalized and therefore always divided by `255.0`.